.github/workflows/CI.yml

```diff
@@ -99,6 +99,8 @@ jobs:
     runs-on: ubuntu-22.04
     container:
       image: "ubuntu:22.04"
+    env:
+      ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@realmarv haven't we agreed in other PRs to call this env GITHUB_TOKEN instead of ACCESS_TOKEN? let's fix it here too

Reply:
Done